pages/2017/index.rst

```diff
@@ -8,4 +8,53 @@ Python in Astronomy 2017
 Lorentz Center, Leiden
 ----------------------
 
-WOOOO YEAG
+The third edition of the Python in Astronomy Conference will be held at the
+Lorentz Center on the **8 - 12th May 2017**.
+
+As with the two previous Python in Astronomy conferences we aim to bring a wide
+variety of people who use, develop or teach people about Python packages in the
+context of all forms of Astronomy. The conference will include presentations,
+tutorials unconference sessions and sprints. As well as building the community
+around astronomical uses of Python the conference aims to improve collaboration
+and interoperability between Python packages, share knowledge on Python packages
+and techniques and provide training and educational materials for users and
+developers of Python packages.
+
+Attending PyAstro17
+###################
+
+Applications for Python in Astronomy 2017 will open on the September 12th 2016,
+participants will be selected both on merit and to ensure a diversity at the
+conference.
+
```

Comment:
@kelle generally I like it. The "people who are new to Python" bit will lead us to getting a lot of applications from people who think it's a training camp (this happened in 2015).